code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1053,13 +1053,16 @@
 
 
 /mob/living/carbon/human/MouseDrop_T(mob/living/target, mob/living/user)
-	if(pulling != target || grab_state != GRAB_AGGRESSIVE || stat != CONSCIOUS || a_intent != INTENT_GRAB)
+	if(pulling != target || stat != CONSCIOUS || a_intent != INTENT_GRAB)
 		return ..()
 
-	//If they can be picked up, and we can't (prevents recursion), try to pick the target mob up as an item.
-	if(HAS_TRAIT(target, TRAIT_HOLDABLE) && !HAS_TRAIT(src, TRAIT_HOLDABLE))
-		if(target.mob_try_pickup(user))
-			return
```

Comment:
Although can we just bump this code below the GRAB_AGGRESSIVE check? It still TECHNICALLY functions and I mean. Come on, it'll be funny.

Reply:
i mean, it'll only really apply to like. mobs that admins apply the holdable trait to, so i dont see why not, but i also dont see why